src/components/molecules/Chatbox/Chatbox.module.scss

```diff
@@ -7,6 +7,8 @@
 }
 
 .chatboxMessages {
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
I don't think depending on CSS is the way to go for displaying data properly. As a rule of thumb, data should be in order even if no styles are applied. 

So, can it be done in a way like `itemsToDisplay.sort().reverse()` or similar before outputing?

I mean, there are even nicer utilities from Flexbox, like `order` CSS property, but if we can display the data properly without resorting to style, I think we should use that.